test/unit/logic/account.js

```diff
@@ -192,15 +181,15 @@ describe('account', function () {
 			account.get({address: validAccount.address}, requestedFields, function (err, res) {
 				expect(err).to.not.exist;
 				expect(res).to.be.an('object');
-				expect(Object.keys(res)).to.eql(requestedFields);
+				expect(Object.keys(res).sort()).to.eql(requestedFields.sort());
 				done();
 			});
 		});
 
 		it('should get all fields if fields parameters is not set', function (done) {
 			account.get({address: validAccount.address}, function (err, res) {
 				expect(err).to.not.exist;
-				expect(Object.keys(res)).to.eql(Object.keys(validAccount));
+				expect(Object.keys(res).sort()).to.eql(Object.keys(validAccount).sort());
```

Comment:
`to.deep.equal` should works without `.sort()`

Reply:
Its because we are comparing arrays. I believe `deep.eql` works with objects.